proxy/proxy/client.go

```diff
@@ -172,7 +182,7 @@ func (c *Client) refreshCfg(instance string) (addr string, cfg *tls.Config, vers
 
 	refreshCfgBuffer := c.RefreshCfgBuffer
 	if refreshCfgBuffer == 0 {
-		refreshCfgBuffer = defaultRefreshCfgBuffer
+		refreshCfgBuffer = DefaultRefreshCfgBuffer
```

Comment:
Since the throttle is at 1 minute, this essentially means that if it doesn't successfully refresh on this attempt, the next refresh will be throttled until the certs expiration. Should we consider lowering the throttle as well? 

For the Java connector, we rate limit at 2 attempts per minute. Maybe we should just match that? WDYT?

Reply:
Matching the Java connector seems like a good idea. I'll update this PR to do that.

Reply:
Fixed in the latest commit.